nservicebus/sagas/index.md

```diff
@@ -110,4 +110,14 @@ Sagas often play the role of coordinator, especially when used in integration sc
 
-The usual way is to correlate on some kind of ID and let the user tell you how to find the correct saga instance using that ID. While this is easily done we decided that this was common enough to warrant native support in NServiceBus for these type of interactions. In V3.0, NServiceBus handles all this for you without getting in your way. If you do `IBus.Reply` in response to a message coming from a saga, NServiceBus will detect it and automatically set the correct headers so that you can correlate the reply back to the saga instance that issued the request. The exception to this rule is the request/response message exchange between two sagas. In such case the automatic correlation won't work and the reply message needs to be explicitly mapped using `ConfigureHowToFindSaga`.
\ No newline at end of file
+The usual way is to correlate on some kind of ID and let the user tell you how to find the correct saga instance using that ID. While this is easily done we decided that this was common enough to warrant native support in NServiceBus for these type of interactions. In V3.0, NServiceBus handles all this for you without getting in your way. If you do `IBus.Reply` in response to a message coming from a saga, NServiceBus will detect it and automatically set the correct headers so that you can correlate the reply back to the saga instance that issued the request. The exception to this rule is the request/response message exchange between two sagas. In such case the automatic correlation won't work and the reply message needs to be explicitly mapped using `ConfigureHowToFindSaga`.
+
+## Querying Saga Data
+Sagas manage state of potentially long-running business processes. When we are building User interfaces that represent those business processes we suddenly feel the urge to query the saga data directly. It can be done, but we recommend to avoid it. It has the following implications:
+
+* The way a given persistence chooses to store the saga data is an implementation detail to the specific persistence that can potentially change over time. By directly querying for the saga data you are coupling that query to this implementation and risk being affected by format changes.
+* By exposing the data outside of the safeguards of the business logic in the saga you risk the data is not treated as read-only. Eventually, a component tries to bypass the saga and directly modify the data. 
+* Querying the data might require additional indexes, resources etc. which need to be managed by the component issuing the query. It can influence saga performance and lead in high contention scenarios to more concurrency conflicts.
```

Comment:
@danielmarbach well the sentence as a whole feel awkward

> It can influence saga performance and lead in high contention scenarios to more concurrency conflicts

do you mean "high contention scenarios" is when accessing data? or when writing data? if writing isnt contention and conflict the same thing?